lib/hunter/commands/hunt.rb

```diff
@@ -173,22 +174,30 @@ def process_packet(data:)
 
         EOF
 
+        buffer = NodeList.load(Config.node_buffer)
+        parsed = NodeList.load(Config.node_list)
+        dest = buffer
+        if node.hostname.match(Regexp.new(@auto_regex))
+          dest = parsed
+          node.label = node.presets[:label] || node.hostname.split(".").first
+        end
+
         if @options.allow_existing || Config.allow_existing
-          buffer.nodes.delete_if { |n| n.id == node.id }
-          buffer.nodes << node
-          puts "Node added to buffer"
+          dest.nodes.delete_if { |n| n.id == node.id }
+          dest.nodes << node
+          puts "Node added to #{File.basename(dest.filepath)} node list"
```

Comment:
I believe `NodeList` already has `NodeList#name` to fetch the name here.

Reply:
Right, could've sworn I looked for a method like that and missed it somehow.